plugins/modules/ntnx_pc_backup_target_v2.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Prem Karat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: ntnx_pc_backup_target_v2
+short_description: Create, Update and Delete a cluster or object store as the backup target.
+version_added: 2.1.0
+description:
+    - Create, Update and Delete a cluster or object store as the backup target.
+    - For a given Prism Central, there can be up to 3 clusters as backup targets and 1 object store as backup target.
+options:
+    state:
+        description:
+            - State of the backup target whether to create, update or delete.
+            - If C(state) is present, it will create or update the backup target.
+            - If C(state) is set to C(present) and ext_id is not provided then it will create a new backup target.
+            - If C(state) is set to C(present) and ext_id is provided then it will update the backup target.
+            - If C(state) is absent, it will delete the backup target.
+        choices: ['present', 'absent']
+        type: str
+    wait:
+        description:
+            - Wait for the task to complete.
+        type: bool
+        required: false
+    domain_manager_ext_id:
+        description: A unique identifier for the domain manager.
```

Comment:
Explain little bit what domain manager is. And how to get this external ID in simple words. Users might not be aware of this `Domain Manager` what we refer to.